.github/labeler.yml

```diff
@@ -1,155 +1,154 @@
 "numpy.ctypeslib.*":
-- changed-files:
-  - any-glob-to-any-file:
-    - src/numpy-stubs/ctypeslib.pyi
+  - changed-files:
+      - any-glob-to-any-file:
+          - src/numpy-stubs/ctypeslib.pyi
 
 "numpy.distutils.*":
-- changed-files:
-  - any-glob-to-any-file:
-    - src/numpy-stubs/distutils/**
+  - changed-files:
+      - any-glob-to-any-file:
+          - src/numpy-stubs/distutils/**
 
 "numpy.dtype":
-- changed-files:
-  - any-glob-to-any-file:
-    - src/numpy-stubs/dtypes.pyi
+  - changed-files:
+      - any-glob-to-any-file:
+          - src/numpy-stubs/dtypes.pyi
```

Comment:
not sure why this is 2/6/10, the indentation is set to '2'. it might be a bug in pretty-yaml

Reply:
you usually prefer [fibonacci indentation](https://github.com/inoric/vscode-fibonacci-indent), right?